WebApiGateway/WebApiGateway.Api/Clients/SubmissionStatusClient.cs

```diff
@@ -81,7 +83,13 @@ public async Task<List<AbstractSubmission>> GetSubmissionsAsync(string queryStri
         {
             await ConfigureHttpClientAsync();
 
-            var response = await _httpClient.GetAsync($"submissions{queryString}");
+            var uriBuilder = new UriBuilder(_httpClient.BaseAddress)
```

Comment:
@Mazar-Shaik Please revert this change and ignore the SQ error

Reply:
Added uriBuilder.Uri.LocalPath instead of uriBuilder.Path

Reply:
please add a unit test